### ISSUE TITLE
open and close a suspended tab to fast with tabs outliner blanks the tab

### ISSUE THREAD

**Issue by alidan** [deanoemcke/thegreatsuspender#1300]
* Extension version: 7.1.8
 * Browser name & version: 87.0.4280.88
 * Operating system & version: win 7 ultimate 64

Ok, this is an issue I found due to accidental opening 50~ tabs I didn't want opened so I closed them quick to bring just one tab up, after doing this, I found that there is a window from green X ing a tab to the tab fully loading where tgs forgets the name of the tab and defaults to something else, essentially completely forgetting the tabs url and giving me a blank. 

I'm not sure how often this comes up in normal us, but I have been hit by it a few times, and because everything still looks like tgs i'm assuming what's happening is a function of tgs and not tabs outliner messing with it. 

tldr, rapidly opening and closing a suspended tab loses the tabs url

**Comment by ossilator** [comment]
there is significant overlap between TGS and Tabs Outliner, so it's not too surprising that they would conflict.

btw, see #1263 - you're vulnerable according to your version number.
